addons/packages/ako-operator/1.4.0/README.md

```diff
@@ -1,14 +1,101 @@
+  membership and fulfill each criterion listed in the Membership Criteria section above.
+
+
+## ako-operator integration with Tanzu Community Edition (TCE)
+
+### Prerequisites
+
+* vSphere is the supported cloud provider for this package. Before proceeding, make sure you have a running management cluster on vSphere
+  following the [steps](https://tanzucommunityedition.io/docs/vsphere-intro/). Verify it is running with `tanzu management-cluster get`.
+* Retrieve the context of the management cluster with `tanzu management-cluster kubeconfig get --admin`, and then switch to the context.
+* Install and set up the AVI Controller on the vCenter Server, following the [documentation](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.4/vmware-tanzu-kubernetes-grid-14/GUID-mgmt-clusters-install-nsx-adv-lb.html#install-avi-controller-on-vcenter-server-7).
+
+### Installation
+
+Fetch a list of available versions.
+
+```shell
+tanzu package available list ako-operator.tanzu.vmware.com -n tkg-system
+```
+
+The ako-operator package with specified version and values can be installed with the Tanzu package CLI.
+Refer to [next section](#configuration-values) for a detailed list of configuration values in `values.yaml`.
+
+```shell
+tanzu package install ako-operator --values-file values.yaml \
+  --package-name ako-operator.community.tanzu.vmware.com \
+  --version 1.5.0 --namespace tkg-system
+```
+
+Verify the ako-operator app has reconciled successfully in the management cluster, by asserting the following grep is non-empty.
+
+```shell
+kubectl get apps -A | grep ako-operator
+```
+
+### Removal
+
+Issue a package delete command with the Tanzu package CLI to remove ako-operator. For example,
+
+```shell
+tanzu package installed delete ako-operator --namespace tkg-system
+```
 
-* ako-operator
 
 ## Configuration
 
-The following configuration values can be set to customize the ako-operator installation.
+### Configuration values
+
+A sample configuration values file `values.yaml` is provided [here](./bundle/config/values.yaml). You should modify this template
+and customize configurations, especially those related to AVI controller credentials.
+Make sure they are correctly configured for the fields including `avi_controller`, `avi_username/password`, `avi_ca_data_b64` etc.
+
+A minimum configuration values file looks like
+
+```yaml
+akoOperator:
+  avi_enable: true
+  namespace: tkg-system-networking
+  config:
+    avi_admin_credential_name: avi-controller-credentials
+    avi_ca_name: avi-controller-ca
+    avi_controller: "10.191.164.223"
+    avi_username: "<avi_username>"
+    avi_password: "<avi password>"
+    avi_cloud_name: "Default-Cloud"
+    avi_service_engine_group: "Default-Group"
+    avi_management_cluster_service_engine_group: "MC-SEG"
+    avi_data_network: "VM Network"
+    avi_data_network_cidr: "10.191.160.0/20"
+    avi_management_cluster_vip_network_name: "VM Network"
```

Comment:
This one is optional, users can use the same network for mgmt and workload clusters, remove it from here.